cmd/create/idp/cmd.go

```diff
@@ -314,26 +335,20 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
-	// Grab all the IDP information interactively if necessary
-	idpType := args.idpType
-	if idpType == "" {
-		interactive.Enable()
-	}
-
```

Comment:
By moving this block, it prevents the `Interactive mode enabled` message from being shown

Reply:
good thing I did this little bit in a separate commit so I just dropped it.